### DISCUSSION TITLE
Anti-Money Laundering Policy

### DISCUSSION THREAD

**Discussion by clementbiron** [OpenTermsArchive/terms-types#40] [Submitted]
### Type name

Anti-Money Laundering Policy

### Alternative type names

AML Policy
Combating the Financing of Terrorism Policy
AML/CFT Policy
AMLCFT Policy

### Writer

service provider

### Audience

employees, directors, officers, contractors, representatives

### Object

risk mitigation of involvement in money laundering or terrorist financing activities

### Examples

| Example | Service Name | URL | Title | Jurisdiction |
|:----|:----|:----|:------|:-------------|
| 1 | Transfermate | [URL 🔗](https://www.transfermate.io/en/anti_money_laundering_policy.html) | ANTI-MONEY LAUNDERING POLICY | ❔ |
| ~2~ | ~FINRA~ | ~[URL 🔗](https://www.finra.org/rules-guidance/key-topics/aml)~ | ~AML~ | ❔ |
| 3 | Goldsmiths | [URL 🔗](https://www.gold.ac.uk/media/docs/finance/Anti-Money-Laundering-Policy.pdf) | Anti-Money Laundering (AML) Policy | 🇬🇧 |
| 4 | Glencore | [URL 🔗](https://www.glencore.com/.rest/api/v1/documents/static/d335a26d-24c9-4269-bd20-84a08fcd9678/Anti-Money-Laundering-Policy_English.pdf) | Anti-Money Laundering Policy | 🇨🇭 |

### References

- Anti-Money Laundering and Countering the Financing of Terrorism Authority (AMLA): https://finance.ec.europa.eu/financial-crime/amla/about-amla_en
- Wikipedia: https://en.wikipedia.org/wiki/Anti%E2%80%93money_laundering
- Bank Secrecy Act: https://www.irs.gov/businesses/small-businesses-self-employed/bank-secrecy-act

**Comment by MattiSG** [comment]
Based on [Wikipedia](https://en.wikipedia.org/wiki/Anti–money_laundering) and the EU [AMLA](https://finance.ec.europa.eu/financial-crime/amla/about-amla_en), I suggest adding the following alternative names:

- Combating the Financing of Terrorism Policy
- AML/CFT Policy
- AMLCFT Policy

**Reply by clementbiron** [reply]
Thank you, I've taken it into account.

**Comment by MattiSG** [comment]
Shorter object proposal: “risk mitigation of involvement in money laundering or terrorist financing activities”.

**Comment by MattiSG** [comment]
The FINRA example (n°2) is wrong: FINRA is a provider of services to support the implementation of the [Bank Secrecy Act](https://www.irs.gov/businesses/small-businesses-self-employed/bank-secrecy-act), it is not a service provider's policy.

**Comment by MattiSG** [comment]
Audience: “employees, directors, officers, contractors, representatives” sounds like a limitative enumeration to me. I would prefer some more descriptive phrasing, such as “any person working directly or indirectly for the service provider”.
